packages/office-ui-fabric-react/src/components/Rating/docs/new/RatingOverview.md

```diff
@@ -0,0 +1,5 @@
+Ratings show people's opinions of a product, helping people make more informed purchasing decisions. People can also rate products they've purchased.
+
+[Sketch toolkit]()
```

Comment:
These are supposed to have links right?

Reply:
There aren't public individual component links available yet. They'll be added later before these docs go live (they're not going live immediately).